script.js

```diff
@@ -0,0 +1,83 @@
+const API_KEY = 'StAXM2VmTgFtwElaUDbGu5uzTNCXS5tH';
```

Comment:
Something is wrong. I guess on serve side - `503` status code

https://user-images.githubusercontent.com/34886027/203266486-2f5ed58a-0845-4f4a-ac24-964f4f356b8c.mov

Reply:
Add an error message to the UI that says "Something happened on the server. Please, try again later"